src/NetAnalyzers/UnitTests/Microsoft.NetFramework.Analyzers/DoNotUseInsecureDtdProcessingXmlDocumentSecureDefaultXmlResolverAnalyzerTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpSecurityCodeFixVerifier<
+    Microsoft.NetFramework.Analyzers.DoNotUseInsecureDtdProcessingAnalyzer,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+
+namespace Microsoft.NetFramework.Analyzers.UnitTests
+{
+    public partial class DoNotUseInsecureDtdProcessingAnalyzerTests
+    {
+        private static DiagnosticResult GetCSharpResultAt(int line, int column)
+        => VerifyCS.Diagnostic(DoNotUseInsecureDtdProcessingAnalyzer.RuleXmlDocumentWithNoSecureResolver).WithLocation(line, column);
+
+        [Fact]
+        public async Task XmlDocument_TargetFx451_ShouldGenerateDiagnostic()
```

Comment:
[cosmetic] Could you use the same test method naming convention across your tests? I prefer this one but I think both are fine. I'll let @mavasani gives his preference.

Reply:
Made the names consistent.